students/kyle_odonnell/Lesson03/mailroom.py

```diff
@@ -0,0 +1,172 @@
+# ------------------------------------------------------------------------ #
+# Title: mailroom.py
+# Description: Assignment for Lesson03
+# KODonnell,10.25.2020,created script
+# ------------------------------------------------------------------------- #
+
+
+# Data -------------------------------------------------------------------- #
+
+# Database for donor information
+DONOR_LIST = [
+    {"Donor Name": "William Gates, III", "Total Given": 653784.49, "Num Gifts": 2, "Average Gift": 326892.24},
+    {"Donor Name": "Mark Zuckerberg", "Total Given": 16396.10, "Num Gifts": 3, "Average Gift": 5465.37},
+    {"Donor Name": "Jeff Bezos", "Total Given": 877.33, "Num Gifts": 1, "Average Gift": 877.33},
+    {"Donor Name": "Paul Allen", "Total Given": 708.42, "Num Gifts": 3, "Average Gift": 236.14},
+    {"Donor Name": "Elon Musk", "Total Given": 50.00, "Num Gifts": 2, "Average Gift": 25.00}
+]
```

Comment:
This works, but really want to stick with lists for this version of mailroom.  We'll get to dictionaries next version.  Want to make sure you have a good command of lists before moving on.